packages/mui-base/src/AutocompleteUnstyled/useAutocomplete.test.js

```diff
@@ -144,6 +144,23 @@ describe('useAutocomplete', () => {
         });
       });
 
+      describe('empty', () => {
+        it('does not call getOptionLabel if filter is empty', () => {
+          let countCallGetOptionLabel = 0;
+          const countGetOptionLabel = (option) => {
+            countCallGetOptionLabel += 1;
+            return getOptionLabel(option);
+          };
```

Comment:
This could be replaced with a `spy` from `sinon`:
```suggestion
          const getOptionLabelSpy = spy(getOptionLabel);
```

Then, in `expect`, validate it with `getOptionLabelSpy.callCount`